src/coreclr/jit/compiler.h

```diff
@@ -676,6 +707,49 @@ class LclVarDsc
         return lvIsMultiRegArg || lvIsMultiRegRet;
     }
 
+private:
+    bool m_addrExposed : 1; // The address of this variable is "exposed" -- passed as an argument, stored in a
```

Comment:
This isn't going to bitwise pack with anything, since there aren't adjacent `bool` bit ops. Is there a reason this can't stay as before? (`unsigned char :1`) I'm not sure if you can change public/private visibility within a packing set; something to check.

Also, which `m_` is a "new standard", it seems like since nothing else in this class is defined with that naming convention, we should stick with what's already here and use the `lv` previx like everything else.

Reply:
> Also, which m_ is a "new standard", it seems like since nothing else in this class is defined with that naming convention, we should stick with what's already here and use the lv previx like everything else.

the other fields in this struct are mostly public so for public `lv` prefix makes sense (if somebody is using grep to find them and not IDE tools) but for private fields I don't see any value in having them. 

>> This isn't going to bitwise pack with anything, since there aren't adjacent bool bit ops. Is there a reason this can't stay as before? (unsigned char :1)

fixed, but with bool instead of char if you don't mind.

Reply:
> fixed, but with bool instead of char if you don't mind.

I thought that `unsigned char :1` doesn't pack with `bool :1` due to the rules of C bitfield packing, or maybe MSVC++ behavior. Can you check? It might be safest to use `unsigned char` just to ensure the greatest likelihood of packing across different compilers.

Reply:
https://godbolt.org/ shows that both clang and MSVC don't have problems with `usnigned int a:1; bool b:1` packing.